docs/content/getting-started/index.md

```diff
@@ -89,22 +89,18 @@ This command:
 - Creates a port-forward from localhost to port 3000 inside the container so you can navigate to the app's frontend UI
 - Streams container logs to your terminal
 
-In your browser you should see the demo app:
+Once the sample application is running, you can access it by opening [http://localhost:3000](http://localhost:3000) in a browser:
 
 {{< image src="demo-screenshot.png" alt="Screenshot of the demo container" width=600px >}}
 <br /><br />
 
-Congrats! You're running your first Radius app. When you're ready to move on to the next step, use <kbd>CTRL</kbd>+ <kbd>C</kbd> to exit the command.
+Congrats, you're running your first Radius app! When you're ready to move on to the next step, use <kbd>CTRL</kbd>+ <kbd>C</kbd> to terminate the application.
```

Comment:
```suggestion
Congrats, you're running your first Radius app! When you're ready to move on to the next step, use <kbd>CTRL</kbd>+ <kbd>C</kbd> to exit the command.
```

This may not be the right framing, feel free to reject this change by committing the suggestion above

Reply:
New wording sounds great, thanks!